dom.bs

```diff
@@ -1217,7 +1217,32 @@ for discussion).
    <a>invoke</a> algorithm.
 
    <li><p>If <var>event</var>'s {{Event/type}} attribute value is not <var>listener</var>'s
-   <b>type</b>, terminate these substeps (and run them for the next <a>event listener</a>).
+   <b>type</b>, check for a legacy type attribute match by running these substeps:
+
+   <ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationend":
```

Comment:
Maybe it's better to make this more DRY and say something like if value is one of "animationend", "animationiteration", "animationstart", "transitionend"...

Let <var>original event type</var> be "animationstart"
Change event type to "webkit" + <var>original event type</var>

Or something like that.